src-tauri/src/northstar/mod.rs

```diff
@@ -65,16 +68,16 @@ pub fn launch_northstar(
     let host_os = get_host_os();
 
     // Explicitly fail early certain (currently) unsupported install setups
-    if host_os != "windows"
-        || !(matches!(game_install.install_type, InstallType::STEAM)
-            || matches!(game_install.install_type, InstallType::ORIGIN)
-            || matches!(game_install.install_type, InstallType::UNKNOWN))
-    {
-        return Err(format!(
-            "Not yet implemented for \"{}\" with Titanfall2 installed via \"{:?}\"",
-            get_host_os(),
-            game_install.install_type
-        ));
+    if host_os != "windows" {
+        if !matches!(game_install.install_type, InstallType::STEAM) {
```

Comment:
Note to self: While this technically fucks with the logic so that now `InstallType::EAPLAY` is now no longer caught as not launchable, this is fine as:

1. EA App is not properly detected anyway as it's caught by the same detection mechanism as Origin and as such  is always misclassified as "Origin"
2. Launching EA App works just fine as long as the location is read/write which is caught by an earlier check during Northstar installation.